ert_shared/ensemble_evaluator/config.py

```diff
@@ -191,7 +191,8 @@ def get_server_ssl_context(
             tempfile.tempdir = backup_default_tmp
 
 
-class EvaluatorServerConfigInfo(BaseModel):
-    dispatch_uri: str
-    token: str
-    cert: str
+class EvaluatorServerConfigInfo(object):
```

Comment:
This isn't Python 2, we don't need `(object)` anymore.

But why remove Pydantic? You can use `from typing import NamedTuple` or `from dataclasses import dataclass` maybe?

Reply:
I was having some problems when unpickling pydantic objects in the forkserver.